packages/ingest/src/granule.ts

```diff
@@ -219,7 +219,7 @@ export async function handleDuplicateFile(params: {
   }) => Promise<void>,
   ACL?: string,
   sourceBucket?: string,
-  fileRemotePath: string,
+  fileRemotePath?: string,
```

Comment:
The changes in this file were all typing fixes related to incorrect/insufficient types